infra/scripts/cometbft.toml

```
@@ -171,4 +171,34 @@ env = { "CONTAINER_NAME" = "${CMT_CONTAINER_NAME}" }
 
 [tasks.cometbft-logs]
 extend = "docker-logs"
-env = { "CONTAINER_NAME" = "${CMT_CONTAINER_NAME}" }
\ No newline at end of file
+env = { "CONTAINER_NAME" = "${CMT_CONTAINER_NAME}" }
+
+[tasks.seed-enable]
+script="""
+sed -i'' -e "s|seed_mode = false|seed_mode = true|" ${CMT_DIR}/config/config.toml
+"""
+
+[tasks.addr-book-enable]
+script="""
+sed -i'' -e "s|addr_book_strict = false|addr_book_strict = true|" ${CMT_DIR}/config/config.toml
+"""
+
+[tasks.set-seeds]
+script = """
+sed -i'' -e 's|^seeds = ""$|seeds = "{{PLACEHOLDER}}"|g' ${CMT_DIR}/config/config.toml
+sed -i "s/{{PLACEHOLDER}}/$BOOTSTRAPS/g" ${CMT_DIR}/config/config.toml
```

This sed script does not work on MAC. Please use the format as above

```suggestion
sed -i'' -e "s/{{PLACEHOLDER}}/$BOOTSTRAPS/g" ${CMT_DIR}/config/config.toml
```

---

Did you tested it in Mac? I will double-check if it works in Linux after the change. I had a really hard time to make this one work.